src/NuGet.Clients/NuGet.PackageManagement.UI/Xamls/PackageManagerControl.xaml.cs

```diff
@@ -140,6 +140,8 @@ private async ValueTask InitializeAsync(PackageManagerModel model, INuGetUILogge
             ApplySettings(settings, Settings);
             _initialized = true;
 
+            _detailModel.IsCentralPackageManagementEnabled = await IsCentralPackageManagementEnabledAsync(CancellationToken.None);
```

Comment:
This seems to be running on UI thread. Consider switching to background thread to avoid UI delays.